browser/ui/views/side_panel/brave_side_panel_coordinator.cc

```diff
@@ -151,8 +151,23 @@ void BraveSidePanelCoordinator::PopulateSidePanel(
   // Notify to give opportunity to observe another panel entries from
   // global or active tab's contextual registry.
   auto* brave_browser_view = static_cast<BraveBrowserView*>(browser_view_);
-  brave_browser_view->WillShowSidePanel();
-
+  CHECK(browser_view_->unified_side_panel()->children().size() == 1);
```

Comment:
In cr132,  we could revert this change because upstream doesn't fallback to global entry after deregestering.
It'll just close deregistered entry and do nothing. So, we don't need this logic anymore.